### DISCUSSION TITLE
Can't figure out how to change IOC prompt.

### DISCUSSION THREAD

**Discussion by Robatronic** [dls-controls/pythonSoftIOC#73] [Q&A] [answered]
I'm trying to customize iocApp prompts similar to:

epicsEnvSet IOCSH_PS1 "AARMS (s71500)> "

How do I accomplish that with pythonSoftIOC....?

Cheers,
Rob

**Comment by Araneidae** [comment, marked as answer]
Don't forget, this isn't an IOC interpreter, it's the standard Python shell.  I did a quick google search, and it appears that `sys.ps1` and `sys.ps2` will do what you want:
```python
$ python
>>> import sys
>>> sys.ps1 = 'Well howdy there? '
Well howdy there? sys.ps2 = '... you have more for me? '
Well howdy there? (
... you have more for me? )
()
Well howdy there? 
```

**Comment by Robatronic** [comment]
Ah!! Of course!!  :)

Thanks so much!!!
